### DISCUSSION TITLE
Metrics-toolkit support for CH2.0

### DISCUSSION THREAD

**Discussion by tgkumarmca** [mulesoft-catalyst/metrics-toolkit#210] [General]
Hi Team,

We are trying to deploy metrics toolkit on CH2.0. we found that the metrics are being captured of the applications deployed in CH1.0. Can someone help me to understand the way to capture metrics of applications deployed in CH2.0.

**Comment by ganesh-integration** [comment]
Hi Team,

metrics-toolkit is not providing the metrics related to cloudhub2.0.
is there a way to retrieve the metrics related to cloudhub2.0

**Comment by ajayshankarj** [comment]
It would be a great addition if this tool can provide metrics related to Cloudhub 2.0
